modules/bastion-host/variables.tf

```
@@ -46,3 +46,9 @@ variable "source_image" {
   default     = "gce-uefi-images/ubuntu-1804-lts"
 }
 
+variable "startup_script" {
+  description = "The script to be executed when the bastion host starts. It can be used to install additional software and/or configure the host."
+  type        = "string"
```

```suggestion
  type        = string
```
@veggiemonk One other thing I missed. Let's make the variable types consistent like the others.

---

Then we're good to ship it.